sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/codegen/CodeFormatter.scala

```diff
@@ -95,9 +95,128 @@ object CodeFormatter {
     new CodeAndComment(code.result().trim(), map)
   }
 
-  def stripExtraNewLinesAndComments(input: String): String = {
```

Comment:
Could you move the existing regex  below into the benchmark code side? I think we don't need this in this class.
```
  val commentHolder = """\/\*(.+?)\*\/""".r
  val commentRegexp =
    ("""([ |\t]*?\/\*[\s|\S]*?\*\/[ |\t]*?)|""" + // strip /*comment*/
      """([ |\t]*?\/\/[\s\S]*?\n)""").r           // strip //comment
  val extraNewLinesRegexp = """\n\s*\n""".r       // strip extra newlines
```

Reply:
sure, moved